server/lomas_server/utils/example_inputs.py

```diff
@@ -0,0 +1,224 @@
+# Dummy queries
```

Comment:
i think it is a duplicate from utils/query_example.py (?) maybe there was an issue during rebasing. otherwise waht is the difference?